renderer/src/components/InterfaceCalls.tsx

```diff
@@ -0,0 +1,7 @@
+export async function getStationUserOnboardingCompleted (): Promise<boolean> {
+  return await window.electron.stationConfig.getOnboardingCompleted()
+}
+
+export async function setStationUserOnboardingCompleted (): Promise<void> {
+  return await window.electron.stationConfig.setOnboardingCompleted()
+}
```

Comment:
I'm not sure about this file, and its placement. It's not a frontend component, is it?

What do you think about moving it to `renderer/src/lib/station-config.tsx`, to mirror the file `main/station-config.js`? And it would be cool to mirror its function names as well (`getOnboardingCompleted` and `setOnboardingCompleted`), to unify the code base.

Reply:
I'm not convinced with that location too. 
However, I'm still not sure about the final aspect of this interfaces, and just placed it in the `components` folder. `renderer/src/lib` seems perfect.
Maybe moving all the api calls to a configuration-like object and using it in the frontend, like `config.setOnboarding`.  Nevertheless, I will move it to `lib`

Reply:
@bajtos what do you think?

Reply:
sorry, forgot about this unresolved comment

Reply:
Sure, b92c4873ce82e1851db6bfc4c6a9867ad425f5ad